api-inference-community/tests/test_dockers.py

```diff
@@ -79,6 +79,19 @@ def test_espnet(self):
             "espnet/kamo-naoyuki_mini_an4_asr_train_raw_bpe_valid.acc.best",
         )
 
+    def test_fasttext(self):
+        self.framework_docker_test(
+            "fasttext",
+            "text-classification",
+            "osanseviero/fasttext_nearest",
```

Comment:
This is OK for tests, but because repos can only have 1 pipeline_tag on the hub, maybe create a duplicate here so we can also test directly on the hub later ?

Reply:
Good point. Now we have https://huggingface.co/osanseviero/fasttext_embedding